dbt/README.md

```diff
@@ -167,16 +184,16 @@ dbt build --select source:pudl.{table_name} --store-failures --target={etl-fast|
 ```
 
 The output of this command should show you a `sql` query you can use to see partitions
-where the row count test failed. To see these, you can do:
+where the row count test failed. While you can copy and paste these queries into
+`duckdb`, this can be quite tedious if many tables have changed, so we've also
+provided a command to [summarize all failures](#summarize-row-count-diffs). Basic usage looks like:
 
 ```bash
-duckdb {PUDL_OUTPUT}/pudl_dbt_tests.duckdb
+dbt_helper summarize-row-count-diffs
 ```
```

Comment:
I would add the `--verbose` flag here because that is the most useful output.

Reply:
If it's the most useful output and we expect it to be the main way that this subcommand is used, maybe it shouldn't require a `--verbose` flag at all.  Is there a good use case for the non-verbose output?